app/api/deps.py

```
@@ -0,0 +1,65 @@
+):
+    client = client_type(student_id=request.student_id, password=request.password)
+    try:
+        await client.login()
+        yield client
+    except ClientException as e:
+        raise HTTPException(status_code=e.status_code, detail=e.detail)
+    finally:
+        await client.close()
+
+
+async def get_kutis_client(request: LoginRequest = Body(...)):
```

Ruff (0.11.9)

Missing return type annotation for public function `get_kutis_client`

(ANN201)